spawningtool/lotv_constants.py

```diff
@@ -65,852 +65,995 @@
+        "display_name": "Protoss Ground Weapons Level 3"
+    },
+    "ProtossGroundArmorsLevel1": {
+        "build_time": 129,
+        "built_from": [ "Forge" ],
+        "display_name": "Protoss Ground Armor Level 1"
+    },
+    "ProtossGroundArmorsLevel2": {
+        "build_time": 154,
+        "built_from": [ "Forge" ],
+        "display_name": "Protoss Ground Armor Level 2"
+    },
+    "ProtossGroundArmorsLevel3": {
+        "build_time": 179,
+        "built_from": [ "Forge" ],
+        "display_name": "Protoss Ground Armor Level 3"
+    },
+    "ProtossShieldsLevel1": {
+        "build_time": 129,
+        "built_from": [ "Forge" ],
+        "display_name": "Protoss Shields Level 1"
+    },
+    "ProtossShieldsLevel2": {
+        "build_time": 154,
+        "built_from": [ "Forge" ],
+        "display_name": "Protoss Shields Level 2"
+    },
+    "ProtossShieldsLevel3": {
+        "build_time": 179,
+        "built_from": [ "Forge" ],
+        "display_name": "Protoss Shields Level 3"
+    },
+    "ProtossAirWeaponsLevel1": {
+        "build_time": 114,
+        "built_from": [ "CyberneticsCore" ],
+        "display_name": "Protoss Air Weapons Level 1"
+    },
+    "ProtossAirWeaponsLevel2": {
+        "build_time": 136,
+        "built_from": [ "CyberneticsCore" ],
+        "display_name": "Protoss Air Weapons Level 2"
+    },
+    "ProtossAirWeaponsLevel3": {
+        "build_time": 157,
+        "built_from": [ "CyberneticsCore" ],
+        "display_name": "Protoss Air Weapons Level 3"
+    },
+    "ProtossAirArmorsLevel1": {
+        "build_time": 114,
+        "built_from": [ "CyberneticsCore" ],
+        "display_name": "Protoss Air Armor Level 1"
+    },
+    "ProtossAirArmorsLevel2": {
+        "build_time": 136,
+        "built_from": [ "CyberneticsCore" ],
+        "display_name": "Protoss Air Armor Level 2"
+    },
+    "ProtossAirArmorsLevel3": {
+        "build_time": 157,
+        "built_from": [ "CyberneticsCore" ],
+        "display_name": "Protoss Air Armor Level 3"
+    },
+    "WarpGateResearch": {
+        "build_time": 100,  # 114s -> 100s in 4.8.2
+        "built_from": [ "CyberneticsCore" ],
+        "display_name": "Warp Gate"
+    },
+    "Charge": {
+        "build_time": 100,
+        "built_from": [ "TwilightCouncil" ],
+        "display_name": "Charge"
+    },
+    "BlinkTech": {
+        "build_time": 121,
+        "built_from": [ "TwilightCouncil" ],
+        "display_name": "Blink"
+    },
+    "AdeptShieldUpgrade": {  # deprecated from LotV Beta
+        "build_time": 57,
+        "built_from": [ "TwilightCouncil" ],
+        "display_name": "Shield Upgrade"
+    },
+    "AdeptPiercingAttack": {
+        "build_time": 100,
+        "built_from": [ "TwilightCouncil" ],
+        "display_name": "Resonating Glaives"
+    },
+    "ObserverGraviticBooster": {
+        "build_time": 57,
+        "built_from": [ "RoboticsBay" ],
+        "display_name": "Observer Gravitic Boosters"
```

Comment:
This is interesting. I appear to have used "Gravitic Boosters" before but also used "Observer Gravitic Boosters" in other places. Do you happen to know off the top of your head which is actually used the SC2 client? I suspect just "Gravitic Boosters"

Reply:
Yes, it is just Gravitic Boosters. I just pushed a commit that fixes that.